tests/curve/test_curve_swap.py

```diff
@@ -0,0 +1,14 @@
+from brownie import interface
+from helpers.addresses import registry as registry_addr
+
+def test_deposit_given_amounts(safe, curve, USDC):
+    TOKEN = interface.ICurveLP(registry_addr.eth.treasury_tokens.crv3pool, owner=safe.account)
+    DAI = interface.ERC20(registry_addr.eth.treasury_tokens.DAI)
+    
```

Comment:
make sure to mint the token using `MintableForkToken` so that the test doesnt rely on actual onchain balances (and thus cannot fail for that reason).